lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -135,6 +135,11 @@ UsdStageWeakPtr getStage(const Ufe::Path& path) { return g_StageMap.stage(path);
 
 Ufe::Path stagePath(UsdStageWeakPtr stage) { return g_StageMap.path(stage); }
 
+PXR_NS::SdfPath getPath(const Ufe::Path& path)
+{
+    return SdfPath(path.getSegments().back().string());
```

Comment:
We should check that the final segment's runTimeId is USD, otherwise we should return an invalid SdfPath, or throw.